.NET/Microsoft.Recognizers.Text.DateTime/French/Extractors/FrenchDateTimeExtractorConfiguration.cs

```diff
@@ -0,0 +1,107 @@
+             new Regex(
+                 DateTimeDefinitions.TimeOfTodayAfterRegex, 
+                 RegexOptions.IgnoreCase | RegexOptions.Singleline);
+
+        public static readonly Regex TimeOfTodayBeforeRegex =
+            new Regex(
+                DateTimeDefinitions.TimeOfTodayBeforeRegex, 
+                RegexOptions.IgnoreCase | RegexOptions.Singleline); 
+
+        public static readonly Regex SimpleTimeOfTodayAfterRegex =
+            new Regex(
+                DateTimeDefinitions.SimpleTimeOfTodayAfterRegex,
+                RegexOptions.IgnoreCase | RegexOptions.Singleline);
+
+        public static readonly Regex SimpleTimeOfTodayBeforeRegex =
+            new Regex(
+                DateTimeDefinitions.SimpleTimeOfTodayBeforeRegex,
+                RegexOptions.IgnoreCase | RegexOptions.Singleline);
+
+        public static readonly Regex TheEndOfRegex = 
+            new Regex(
+                DateTimeDefinitions.TheEndOfRegex,
+                RegexOptions.IgnoreCase | RegexOptions.Singleline);
+
+        public static readonly Regex UnitRegex = 
+            new Regex(
+                DateTimeDefinitions.TimeUnitRegex,
+                RegexOptions.IgnoreCase | RegexOptions.Singleline);
+
+        public FrenchDateTimeExtractorConfiguration()
+        {
+            DatePointExtractor = new BaseDateExtractor(new FrenchDateExtractorConfiguration());
+            TimePointExtractor = new BaseTimeExtractor(new FrenchTimeExtractorConfiguration());
+            DurationExtractor = new BaseDurationExtractor(new FrenchDurationExtractorConfiguration());
+            UtilityConfiguration = new FrenchDatetimeUtilityConfiguration();
+        }
+
+        public IExtractor DatePointExtractor { get; }
+
+        public IExtractor TimePointExtractor { get; }
+
+        public IDateTimeUtilityConfiguration UtilityConfiguration { get; }
+
+        Regex IDateTimeExtractorConfiguration.NowRegex => NowRegex;
+
+        Regex IDateTimeExtractorConfiguration.SuffixRegex => SuffixRegex;
+
+        Regex IDateTimeExtractorConfiguration.TimeOfTodayAfterRegex => TimeOfTodayAfterRegex;
+
+        Regex IDateTimeExtractorConfiguration.SimpleTimeOfTodayAfterRegex => SimpleTimeOfTodayAfterRegex;
+
+        Regex IDateTimeExtractorConfiguration.TimeOfTodayBeforeRegex => TimeOfTodayBeforeRegex;
+
+        Regex IDateTimeExtractorConfiguration.SimpleTimeOfTodayBeforeRegex => SimpleTimeOfTodayBeforeRegex;
+
+        Regex IDateTimeExtractorConfiguration.TimeOfDayRegex => TimeOfDayRegex;
+
+        Regex IDateTimeExtractorConfiguration.TheEndOfRegex => TheEndOfRegex;
+
+        Regex IDateTimeExtractorConfiguration.UnitRegex => UnitRegex;
+
+        public IExtractor DurationExtractor { get; }
+
+        public bool IsConnector(string text)
+        {
+            return (string.IsNullOrEmpty(text) || text.Equals(",") ||
+                        PrepositionRegex.IsMatch(text) || text.Equals("t") || text.Equals("pour") ||
```

Comment:
What does "t" mean here?
Can you add an example as comment in code?

Reply:
This is a possible point for refactor as the list of values probably should be in the yaml file.